docs/explanations/which_services.rst

```diff
@@ -0,0 +1,188 @@
+         vulnerabilities while the system runs. It removes the need to
+         immediately reboot after a kernel upgrade so that you can schedule
+         downtime when it's convenient. This tooling is available through
+         ``livepatch`` and is one of the services we enable by default.
+
+         * Find out more about what Livepatch can do for your organisation in
+           the `Livepatch documentation`_, or learn
+           :ref:`how to enable it<manage-livepatch>` with ``pro``.
+
+       * **Landscape**
+
+         `Landscape`_ is the leading management and administration tool for
+         Ubuntu. It can manage up to 40,000 Ubuntu machines with a single
+         interface, and can automate security patching, as well as hardening
+         and compliance. Landscape can be `self-hosted`_ or you can use Canonical's
+         `Landscape SaaS`_ offering.
+
+         * Find out more about
+           :ref:`registering a machine with Landscape<manage-landscape>`
+           using ``pro``.
+
+       Want more information?
+
+    .. tab-item:: Security certification
+
+       Ubuntu provides security `compliance, certifications, and hardening <certifications_>`_:
+
+       * **FIPS 140-2**
+
+         * `FIPS 140-2 Certified Modules <fips_>`_ are available through ``fips``.
+
+         * Compliant but non-certified patches are available through
+           ``fips-updates``.
+
+         * Find out more :ref:`about managing FIPS<manage-fips>`.
+
+       * **Ubuntu Security Guide (USG)**
+
+         * In the 20.04 LTS we introduced the `Ubuntu Security Guide <usg_>`_, which
+           provides security tooling through ``usg``. It bundles together
+           multiple key components, such as CIS benchmarking and DISA-STIG.
+
+         * Before USG, Center for Internet Security `(CIS) Benchmark <cis_>`_ tooling
+           was available as a separate service through ``cis``. This can be
+           used up to (and including) 20.04 LTS, but on all later LTS releases
+           this functionality is provided through ``usg``.
+
+         * Find out :ref:`how to manage USG (and CIS)<manage-cis>`
+           with the Pro Client.
+
+       * **Common Criteria**
+
+         * `Common Criteria EAL2 (CC EAL) <CC_>`_ certification tooling is available
+           through ``cc-eal``. The Ubuntu 18.04 LTS and 16.04 LTS have both
+           been certified.
+
+         * Find out :ref:`how to enable CC EAL<manage-cc>` on your LTS.
+
+       Want more information?
+
+    .. tab-item:: Personal user
+
+       * **Expanded Security Maintenance (ESM)**
+
+         If you're using Pro on an LTS machine (Desktop or Server) you can get
+         additional security support through
+         `Expanded Security Maintenance (ESM) <esm_>`_. We recommend it for everyone
+         using Pro, and it's enabled by default. It includes two services:
+
+         * ``esm-infra`` provides security updates for packages in the Ubuntu
```

Comment:
We are using the same description here as the one on the commercial tab. Maybe we can create a template for it and reuse it on each tab that we need.

I am thinking of an approach like this one:
https://github.com/canonical/ubuntu-pro-client/blob/docs/docs/tutorials/fix_intro.rst?plain=1#L7

This idea also applies to any other section were we are reusing the same description for a service

Reply:
That's a really good point, but I think I'll consider that out of scope for this PR to avoid inflating it further. At some point I'll want to do a general review of every page and modularise everything that can be modularised - it will help to make it easier for other teams to sync the content, now that I know how to do it.